### PR TITLE
Replace json with jsonc highlighting

### PR DESCRIPTION
**Pull request by Andre601**
Just changed the `json` highlighting to `jsonc` (JSON with comments) to get rid of those annoying, ugly red error highlighting.
The downside is, that the displayed code looks a bit more... bland now (i.e. keys are now a light blue instead of green), but I consider this a minor downside for a greater good, that being better readability.

### CODE REVIEW

**Comment by Kir-Antipov** [comment]
Oh, I'm sorry, but I cannot accept this PR, because I already rejected a similar one in the past (#12). The problem is users tend to copy-paste everything written in README, and I don't blame them, that's what examples are for, but Fabric Loader **does not** accept JSON5 or JSONC as a valid metadata format, it will throw `net.fabricmc.loader.impl.lib.gson.MalformedJsonException` in case you leave a comment in your `fabric.mod.json`. Therefore, I consciously use `json` in README, so readers are able to easily see that comments cannot be declared in their metadata.

If you want to help the project, you can add metadata readers for plugins :)
I do not develop them, therefore I know nothing about their configurations, but it would be handy to have support for such things for users like you :)

Example: [`Implemented Quilt support`](https://github.com/Kir-Antipov/mc-publish/commit/0fcfdc07c1fe370e95e1d50a523ef60fe4c2cecb)

**Comment by Andre601** [comment]
That's understandable.

Regarding the plugin thing. Idk if I can provide some real contribution outside of some info here (JS and TS are not languages I work with).

Plugins for Servers like Spigot, Paper or any fork of those use a `plugin.yml` to define values for the server to read and use.
Meanwhile, proxies like BungeeCord, Waterfall and forks also use a `plugin.yml` for the same goals. But they also support and actually prioritize a `bungeey.yml` (This was afaik made to allow hybrid plugins that can run on both platforms. Both have different APIs so this was most likely made to avoid major conflicts).

Now, Velocity is a different book. It uses a `velocity-plugin.json` file.
This file is either created at compile-time using a `@Plugin` annotation or manually by the plugin author.

I can't say if Velocity is very strict on what values are allowed in a config (i.e. if it ignores unknown keys or gives warnings about those), but I can say that spigot plugins and bungee plugins do not care as long as their required values (main, author(s), version, name) are present.

So, tl;dr:

- Spigot, Paper and forks of those use a `plugin.yml` file for plugin settings
- BungeeCord, Waterfall and forks of those also use a `plugin.yml` file but will use and prioritize a `bungee.yml` file if present, so the action should do the same in this case.
- Velocity uses a JSON file which is either auto-created through annotations or manually made by the plugin author.

Example files from my plugin:
- [bungee.yml](https://github.com/Andre601/AdvancedServerList/blob/master/bungeecord/src/main/resources/bungee.yml)
- [plugin.yml](https://github.com/Andre601/AdvancedServerList/blob/master/paper/src/main/resources/plugin.yml)
- [velocity-plugin.json](https://github.com/Andre601/AdvancedServerList/blob/master/velocity/src/main/resources/velocity-plugin.json)
